### PR TITLE
sermon detail view additions

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: src/components/organisms/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+						<FormattedMessage
+							id={`footer__NavItemGerman`}
+							defaultMessage="Deutsch"
+							description={`Footer nav link name: German`}
+						/>
+					</a>
+				</li>
+				<li>
+					<a href="/en">
+						<FormattedMessage
```
Comment (mattleff) [inline src/components/organisms/footer.tsx:213]: These don't need to be translated, right? We always show the language name in that language?

File: src/components/organisms/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+					</a>
+				</li>
+			</ul>
+
+			<a
+				href="https://itunes.apple.com/us/app/audioverse/id726998810?mt=8"
+				target={'_blank'}
+				rel={'noreferrer noopener'}
+			>
+				<img src="/img/app-apple.png" alt="iOS App" />
```
Comment (mattleff) [inline src/components/organisms/footer.tsx:272]: i18n the alt text?

File: src/components/organisms/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+			<a
+				href="https://play.google.com/store/apps/details?id=org.audioverse.exodus"
+				target={'_blank'}
+				rel={'noreferrer noopener'}
+			>
+				<img src="/img/app-android.png" alt="Android App" />
+			</a>
+
+			<p>
+				<span>{`AudioVerse © ${new Date().getFullYear()} All Rights Reserved.`}</span>{' '}
```
Comment (mattleff) [inline src/components/organisms/footer.tsx:284]: i18n here and below

File: src/components/organisms/header.tsx [hunk 4]
```diff
@@ -1,15 +1,206 @@
...
+	const languageRoute = useLanguageRoute();
+
+	return (
+		<header className={styles.header}>
+			<ul>
+				<li>
+					<a
+						href={`https://audioversestore.org/`}
+						target={'_blank'}
+						rel={'noreferrer noopener'}
```
Comment (mattleff) [inline src/components/organisms/header.tsx:19]: Is there a reason we're using `noreferrer noopener` for AV store and JU, but not using them for T&S up in footer.tsx?

File: src/components/organisms/header.tsx [hunk 5]
```diff
@@ -1,15 +1,206 @@
...
+							id={`header__sisterSiteJourneysUnscriptedLink`}
+							defaultMessage="Journeys Unscripted"
+							description={`Header sister site link: Journeys Unscripted`}
+						/>
+					</a>
+				</li>
+			</ul>
+			<h1>
+				<a href={`/${languageRoute}`}>
+					<img src="/img/av-logo.png" alt="AudioVerse" width={300} />
```
Comment (mattleff) [inline src/components/organisms/header.tsx:44]: Should this size be set in CSS?

File: src/components/organisms/header.tsx [hunk 6]
```diff
@@ -1,15 +1,206 @@
...
+					<a href={`/${languageRoute}/sermons/all/page/1`}>
+						<FormattedMessage
+							id={`header__navItemPresentations`}
+							defaultMessage="Presentations"
+							description={`Header nav link name: Presentations`}
+						/>
+					</a>
+				</li>
+				<li>
+					<a href="#">
```
Comment (mattleff) [inline src/components/organisms/header.tsx:67]: Might be good to leave a TODO here for all these links.

File: src/components/organisms/recording.tsx [hunk 7]
```diff
@@ -256,6 +256,58 @@ export function Recording({ recording }: RecordingProps): JSX.Element {
...
+			<p>{recording.shareUrl}</p>
+			<label>
+				<FormattedMessage
+					id="sermonDetailPage__embedCodeLabel"
+					defaultMessage="Embed Code"
+					description="Sermon detail embed code label"
+				/>{' '}
+				<input
+					readOnly={true}
+					value={`<iframe src="https://www.audioverse.org/english/embed/media/${recording.id}" width="500" height="309" scrolling="no" frameBorder="0" ></iframe>`}
```
Comment (mattleff) [inline src/components/organisms/recording.tsx:282]: Do we have this path on our task list/maintain URL backward-compatibility?

File: src/pages/[language]/sermons/[id].ts [hunk 8]
```diff
@@ -27,7 +28,7 @@ export async function getStaticProps({
 		props: {
 			sermon,
 		},
-		revalidate: 10,
+		revalidate: REVALIDATE,
```
Comment (mattleff) [inline src/pages/[language]/sermons/[id].ts:31]: Nice to have this in a const!

**Review by mattleff — APPROVED** [review]
Tons of great work here! It's starting to come together! 🍒

File: src/components/organisms/header.tsx (hunk 5 above)
Comment (narthur) [reply src/components/organisms/header.tsx:44]: No I don't think so. The size should be constrained even without or before CSS loading.

**Review by mattleff — APPROVED** [review]
LGTM!

File: src/components/organisms/recording.tsx (hunk 7 above)
Comment (narthur) [reply src/components/organisms/recording.tsx:282]: Yes, it's on my task list :)

**Pull request merged**